src/inmanta/data/dataview.py

```diff
@@ -1315,15 +1315,26 @@ def get_base_query(self) -> SimpleQueryBuilder:
         return query_builder
 
     def construct_dtos(self, records: Sequence[Record]) -> Sequence[dict[str, str]]:
-        return [
-            model.DiscoveredResource(
-                discovered_resource_id=res["discovered_resource_id"],
-                values=json.loads(res["values"]),
-                managed_resource_uri=f"/api/v2/resource/{res['discovered_resource_id']}" if res["managed"] else None,
-                discovery_resource_id=res["discovery_resource_id"],
-            ).model_dump()
-            for res in records
-        ]
+        out = []
+        for res in records:
+            if res["discovery_resource_id"]:
```

Comment:
I don't see the value in this, see [here](https://github.com/inmanta/inmanta-core/pull/7912#discussion_r1709054592). I think we can revert this entire block to what it was in db7e7ca.